services/application.js

```diff
@@ -26,7 +26,13 @@ class Application {
     async getApplication(params, context) {
         verifySession(context)
             .verifyInitialized();
-        return await this.getApplicationById(params._id);
+        const application = await this.getApplicationById(params._id);
+        const isAdminOrFedLead = [USER.ROLES.ADMIN, USER.ROLES.FEDERAL_LEAD].includes(context.userInfo?.role);
+        const isSubmitter = application?.applicant?.applicantID === context?.userInfo?._id;
+        if (!isAdminOrFedLead && !isSubmitter){
```

Comment:
I think it would be better to revert the logic, so that only submitter, admin or fed lead can see an application, everything else is blocked by default, this way will be a little more secure.